kubetest/extract_test.go

```diff
@@ -83,3 +83,58 @@ func TestGetKube(t *testing.T) {
 		}
 	}
 }
+
+func TestExtractStrategies(t *testing.T) {
+	cases := []struct {
+		option        string
+		expectURL     string
```

Comment:
Yes, please remove. Code that doesn't exist doesn't have to be maintained. And the person who needs this functionality can always add it (and would probably wind up changing it anyway).

Test-infra has this luxury because we release whenever we want and do not need to support old versions.